src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -10,15 +10,16 @@ import { Card } from "../card";
 import * as styles from './more-blogs-section.module.scss';
 
 const MoreBlogsSection = ({ data, refMoreFrom, title }) => {
-  const { edges } = data.allMdx;
+  const { relatedPosts } = data.mdx;
 
   return (
     <div className={styles.moreFromXlContainer} ref={refMoreFrom} >
       <div className={styles.titleContainer}>
-        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>{`${title} →`}</Link>
+        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>More from about <span>{title}</span>→</Link>
       </div>
       <div className={styles.blogsContainer}>
-        {edges.map(({node}) => <Card data={node} key={node.id} className={styles.cardStyles} />)}
+        {relatedPosts.forEach((node) => <Card data={node} className={styles.cardStyles} />)}
+        {relatedPosts.slice(0, 3).map((post) => <Card data={post} key={post.frontmatter.title} className={styles.cardStyles} />)}
```

Comment:
Why this?

Reply:
Are you referring to the slice? The slice cuts the array to have the three cards I need because the method for creating these cards does not allow a limit.

Reply:
Done